federated-server/utils/faltToUnique.js

```diff
@@ -1 +1 @@
-export const flatToUnique = (prev,act) => [...prev, ...act.filter(node => prev.indexOf(node ) === -1)]
+export const flatToUnique = (prev=[],act) => [...prev, ...act.filter(node => prev.indexOf(node ) === -1)]
```

Comment:
s/faltToUnique/flatToUnique/g in filename

Reply:
done